test/TestConfig/resources/excludes/latest_exclude_SE100.txt

```diff
@@ -25,4 +25,7 @@
 org.openj9.test.vm.Test_MsgHelp:test_loadMessages_EN																	AN-https://github.ibm.com/runtimes/test/issues/46 generic-all
 org.openj9.test.java.lang.management.TestRuntimeMXBean:testGetMBeanInfo													1626 generic-all
 org.openj9.test.java.lang.management.TestThreadMXBean:testGetMBeanInfo													1626 generic-all
-org.openj9.test.vmArguments.VmArgumentTests:testCrNocr																	244 generic-all
\ No newline at end of file
+org.openj9.test.vmArguments.VmArgumentTests:testCrNocr																	244 generic-all
+# Exclude this while investigating https://github.com/eclipse/openj9/issues/2047
+# Intermittent NullPointerException in TestAttachErrorHandling.deleteIpcDirectories
+org.openj9.test.attachAPI.TestAttachErrorHandling:test_startupShutdownRobustness
```

Comment:
missing issue number and platform. We can remove line `Exclude this while investigating https://github.com/eclipse/openj9/issues/2047`